src/ReportManager/SchemaFormContext/SchemaForm/index.test.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+
+import { render, screen, within } from '../../test-utils';
+
+import SchemaForm from './';
+
+describe('ReportManager - SchemaForm', () => {
+
+  const efb = {
```

Comment:
This doesn't feel like a proper name for this variable. Why not `schema` since that is what it is?